app/views/content/landing/how-to-become-a-teacher/_content.html.erb

```diff
@@ -0,0 +1,57 @@
+<div class="row">
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What qualifications do I need?</h2>
+
+    <div class="inset">
+      <p>
+        You'll need to train for qualified teacher status (QTS) to be able to teach in most primary and secondary schools in England.
+      </p>
+
+      <p>
+        If you have a degree or equivalent qualification, you can do postgraduate primary or secondary teacher training.
+        If you do not have a degree and are not studying for one, you can do undergraduate teacher training.
+      </p>
+
+      <p>
+        <%= link_to("Check the qualifications you need to teach", page_path("is-teaching-right-for-me/qualifications-you-need-to-teach")) %>.
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Can I get help funding my training?</h2>
+
+    <div class="inset">
+      <p>
+        Undergraduate and postgraduate teacher training course fees are around £9,250.
+        You can apply for tuition fee and maintenance loans, even if you already have a student loan.
+      </p>
+
+      <p>
+        If you're interested in teaching certain subjects, you might be able to get a scholarship or bursary of up to £29k tax-free.
+        This is to support you while you are training and doesn't need to be paid back.
```

Comment:
```suggestion
        This is to support you while you are training and does not need to be paid back.
```
getting rid of contraction as per GDS style guide https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#contractions